pubspec.yaml

```diff
@@ -11,9 +11,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  rxdart: ^0.22.0
-  google_maps_webservice: ^0.0.14
-  http: ">=0.11.0 <1.0.0"
+  rxdart: 0.22.0
```

Comment:
Too restrictive
I don't think you need to change version here

Reply:
Fixed it!